src/blockchain/node/HeaderOracle.cpp

```diff
@@ -32,6 +32,7 @@
 #include "opentxs/network/p2p/Data.hpp"
 #include "opentxs/util/Container.hpp"
 #include "opentxs/util/Log.hpp"
+#include "util/threadutil.hpp"
```

Comment:
No real changes here

Reply:
True, it's a leftover from some diagnostics, I'll take it out.

Reply:
These times can be change to fine tune the performance, we are not relying on delay timing to avoid race conditions. Race conditions were dealt with by design changes, including an individual reactor thread (processing loop) for each Actor or Worker.